config/locales/en/placements/support/providers/users.yml

```
@@ -12,12 +12,12 @@ en:
                 name: Name
                 email: Email
           remove:
-            page_title: Are you sure you want to remove this user? - %{user_name} - %{organisation_name}
-            are_you_sure: Are you sure you want to remove this user?
-            remove_user: Remove user
+            page_title: Are you sure you want to delete this user? - %{user_name} - %{organisation_name}
+            are_you_sure: Are you sure you want to delete this user?
+            delete_user: Delete user
             cancel: Cancel
-            user_will_be_sent_an_email: The user will be sent an email to tell them you removed them from %{organisation_name}
+            user_will_be_sent_an_email: The user will be sent an email to tell them you deleted them from %{organisation_name}
           show:
-            remove_user: Remove user
+            delete_user: Delete user
           destroy:
-            user_removed: User removed
+            user_removed: User deleted
```

```suggestion
            user_deleted: User deleted
```
Might also need changing in the controller.